packages/host/app/components/operator-mode/container.gts

```diff
@@ -493,9 +507,12 @@ export default class OperatorModeContainer extends Component<Signature> {
     return this.operatorModeStateService.state?.stacks.flat() ?? [];
   }
 
-  get cardForCodeMode() {
-    // Last card in rightmost stack
-    return this.allStackItems.reverse()[0].card;
+  get lastCardInRightMostStack(): CardDef {
+    return (
+      this.allStackItems
+        // @ts-ignore Property 'card' does not exist on type 'StackItem'. - it actually does exist because we filtered for it in the line above
+        .reverse()[0].card
```

Comment:
reverse mutates an array in place. would strongly advise not to use this unless you really wanna mutate this array.

rather, just `this.allStackItems[this.allStackItems.length - 1]`